### ISSUE TITLE
Why are sensible defaults in satpy not the defaults in pycoast?

### ISSUE THREAD

**Issue by mraspaud** [pytroll/pycoast#34]
Why are these sensible defaults not the defaults in pycoast?

_Originally posted by @djhoese in https://github.com/pytroll/satpy/pull/934_

**Comment by djhoese** [comment]
@gerritholl Good catch. I've updated the description with more information. Let me know if that makes sense.
